<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBAcousticComponentEditWidget</class>
 <widget class="QWidget" name="SVDBAcousticComponentEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="masterLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelDisplayName">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelUValue">
     <property name="text">
      <string>Impact sound value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditISV">
     <property name="toolTip">
      <string>U-value is being calculated using the heat conduction values from the assigned boundary conditions.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="labelUValue_6">
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelUValue_5">
     <property name="text">
      <string>Air sound restistence value:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditASRV">
     <property name="toolTip">
      <string>U-value is being calculated using the heat conduction values from the assigned boundary conditions.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QtExt::ColorButton" name="pushButtonColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="labelUValue_2">
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
